comp/forwarder/defaultforwarder/internal/retry/http_transactions_serializer_test.go

```diff
@@ -108,7 +108,7 @@ func TestHTTPTransactionSerializerMissingAPIKey(t *testing.T) {
 func TestHTTPTransactionFieldsCount(t *testing.T) {
 	tr := transaction.HTTPTransaction{}
 	transactionType := reflect.TypeOf(tr)
-	assert.Equalf(t, 11, transactionType.NumField(),
+	assert.Equalf(t, 12, transactionType.NumField(),
```

Comment:
Do we need to do anything relating to the error message that is tied to this assertion? (specifically the bit about updating the HTTP transaction serializer)